kroma-validator/guardian.go

```diff
@@ -496,7 +496,7 @@ func (g *Guardian) checkConfirmCondition(transactionId *big.Int, outputIndex *bi
 		return false, nil
 	}
 
-	isConfirmed, err := g.isTransactionConfirmed(transactionId)
+	isConfirmed, err := g.isTransactionConfirmed(ctx, transactionId)
```

Comment:
please add a sub context with network timeout

Reply:
In the function `isTransactionConfirmed`, there is a declaration of new context with timeout.